dev-docs/publisher-api-reference/setConfig.md

```diff
@@ -472,6 +472,7 @@ The `s2sConfig` properties:
 | `endpoint` | Required | URL or Object | Defines the auction endpoint for the Prebid Server cluster.  See table below for object config properties. |
 | `syncEndpoint` | Required | URL or Object | Defines the cookie_sync endpoint for the Prebid Server cluster. See table below for object config properties. |
 | `userSyncLimit` | Optional | Integer | Max number of userSync URLs that can be executed by Prebid Server cookie_sync per request.  If not defined, PBS will execute all userSync URLs included in the request. |
+| `syncTimeout` | Optional | Integer | Maximum number of milliseconds allowed for each userSync to load |
```

Comment:
The default behaviour is "wait infinitely" right? Maybe this should appear in the docs as well :smile: